public/pages/Correlations/containers/CreateCorrelationRule.tsx

```diff
@@ -56,6 +69,7 @@ export interface CreateCorrelationRuleProps extends DataSourceProps {
     { rule: CorrelationRuleModel; isReadOnly: boolean }
   >['history'];
   notifications: NotificationsStart | null;
+  services: BrowserServices;
```

Comment:
Get the services you need instead of all the services.

Reply:
addressed